spec/models/nomis/offender_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
```

Comment:
On the subject of tests – it might be nice to also cover this in a feature test somewhere it ends up being shown to users. Hopefully there's already a feature test which could be added to.

What do you think @starswan?

Reply:
I didn't feel it was needed in this case, as all this does is change the scenarios where 'N/A' is shown to the user. Our current tests are dominated by feature tests which is less than ideal, as often they break in unfathomable ways which makes testing/debugging much harder than it needs to be.